corehq/apps/locations/management/commands/delete_locations.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.domain.models import Domain
+from corehq.apps.locations.models import Location
+from .check_loc_types import locs_by_domain
+
+
+class Command(BaseCommand):
+    args = "<domain>"
+    help = ("Change the doc types of all locations in `domain` to "
+            "Location-DELETED.  Note that this only affects couch locations.")
+
+    def bulk_delete_locs(self, loc_ids, total):
+        locs_to_save = []
+        count = 0
+        for loc in iter_docs(Location.get_db(), loc_ids):
+            loc['doc_type'] = "{}-DELETED".format(loc['doc_type'])
+            locs_to_save.append(loc)
+            count += 1
+
+            if len(locs_to_save) > 100:
+                Location.get_db().bulk_save(locs_to_save)
```

Comment:
(unrelated to above thread - if this is a huge number of locations I'm a little concerned about DB load. maybe we can add a sleep here?)